pkg/acsengine/defaults-kubelet.go

```diff
@@ -35,7 +35,6 @@ func setKubeletConfig(cs *api.ContainerService) {
 		"--cluster-domain":                  "cluster.local",
 		"--network-plugin":                  "cni",
 		"--pod-infra-container-image":       cloudSpecConfig.KubernetesSpecConfig.KubernetesImageBase + KubeConfigs[o.OrchestratorVersion]["pause"],
-		"--max-pods":                        strconv.Itoa(DefaultKubernetesKubeletMaxPods),
```

Comment:
Seems like DefaultKubernetesKubeletMaxPods isn't used anywhere in the code anymore? If that's the case we should remove it from const.go

Reply:
Yes, I will remove it.